.gitignore

```diff
@@ -1,2 +1,5 @@
 *~
 *.pyc
+*.o
+*.mod
+*.exe
```

Comment:
It occurs to me you meant to add a new line after the *.pyc ignore. I've done so.